packages/auth/test/helpers/integration/helpers.ts

```diff
@@ -113,5 +113,6 @@ export function getTotpCode(
   return token;
 }
 export const email = 'totpuser-donotdelete@test.com';
+export const fakePassword = 'password';
```

Comment:
nit: call this "password" since it is indeed the correct password for this account (though not a recommended one to use in general)